<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="38573" started-at="2019-05-31T12:12:02Z" finished-at="2019-05-31T12:12:41Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(cucumber.api.testng.PickleEventWrapper, cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runner.TestRunner@398b7c]" name="runScenario" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="38573" started-at="2019-05-31T12:12:02Z" finished-at="2019-05-31T12:12:41Z">
      <class name="runner.TestRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:runner.TestRunner@398b7c]" name="setUpClass" is-config="true" duration-ms="30933" started-at="2019-05-31T12:12:02Z" finished-at="2019-05-31T12:12:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="runScenario(cucumber.api.testng.PickleEventWrapper, cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runner.TestRunner@398b7c]" name="runScenario" duration-ms="7540" started-at="2019-05-31T12:12:33Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2019-05-31T12:12:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["MW-17292:Verify OVERLAP NANO software update"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Nano"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at stepdefs.BaseStepDefs.iLoggedInToPortalAndGoToTab(BaseStepDefs.java:53)
at âœ½.I logged in to Portal and go to "NANO" tab(resources\Feature\NanoPortal.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:runner.TestRunner@398b7c]" name="tearDownClass" is-config="true" duration-ms="21" started-at="2019-05-31T12:12:41Z" finished-at="2019-05-31T12:12:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
